### DISCUSSION TITLE
Stolen name?

### DISCUSSION THREAD

**Discussion by Fred-IV** [ciderapp/Cider#1754] [General]
Cider is actually a program to run windows applications on the Mac.
Don't people google before they start using names?

**Comment by cryptofyre** [comment]
We've used this name for quite a while, and besides its multiplatform and not just a Mac app and if anything, we've done more with the name.

**Reply by polymo1** [reply]
"we've done more with the name."
so if I were to do a lot of stuff with any name for a project, I'm just allowed to take it? theres no issue with it?

**Reply by freehelpdesk** [reply]
No need to be arrogant. Kinda like saying its illegal for Apple, and Apple Records to share the same name, or Dove, and Dove Chocolate to share one.

Not only that, Cider is not trade marked.

That is all.

**Comment by cryptofyre** [comment]
Considering the project originally with the name was abandoned and we’ve never heard of it before there’s nothing wrong with us using this name.On Aug 9, 2023, at 9:14 PM, Wes Bryie ***@***.***> wrote:﻿
"we've done more with the name."
so if I were to do a lot of stuff with any name for a project, I'm just allowed to take it? theres no issue with it?
